.nvmrc

```diff
@@ -0,0 +1 @@
+8
```

Comment:
Should we specific the exact version, `8.12.0`? Not sure what the best practice is for nvmrc files

Reply:
When you specify `8` like this, it will default to the latest stable release (in this case, it defaults to `8.12`). I think that's okay given that `package.json` matches up with saying the Node version is `8.x`

@elizabethsjudd what do you think?

Reply:
WH has never ran in to issues by not specify the minor version.  And I agree with @jendowns that if the `package.json` file says `8.x` then just specifying `8` here would be accurate.